configuration/scripts/options/set_nml.ml

```diff
@@ -0,0 +1,7 @@
+
+oceanmixed_ice = .true.
+ocn_data_type = 'ncar'
+ocn_data_format = 'nc'
+ocn_data_dir    = 'ICE_MACHINE_INPUTDATA/CICE_data/forcing/gx1'
+oceanmixed_file = 'pop_frc.gx1.nc'
```

Comment:
Need have a standard name for the oceanmixed_file and then commit to Zenodo.

Reply:
How about ocean_forcing_gx1.nc?

Reply:
It should be named similarly to the atmospheric forcing, but that is not standardized.  It could follow the JRA tx1 example,
CESM_ocean_monthly_forcing_gx1.nc -- probably overkill since its location in the forcing directory structure would identify it:  CICE_data/forcing/gx1/CESM/MONTHLY/
So ocean_forcing_gx1.nc works.

Reply:
@eclare108213 mentioned having 2D in the name as well. Then if we put it under the forcing/gx1 area, we shouldn't need the "gx1" label.

Reply:
I think we want a filename that doesn't rely on it's directory location to describe it.  We also don't want the same filename in multiple directories.  So we want to avoid having multiple "ocean_forcing.nc" files in multiple directories.  When the file is separated from it's directory, it should be unique (relative to other input data files) and still somewhat descriptive.  I am completely fine with something like "CESM_ocean_monthly_climatology_forcing_gx1_210324.nc".  But it need not be that descriptive.  It's always a balance.  When/if we update this file in the future, we'll need to give it a new name as well.  We need to make sure we never have multiple versions of the same filename, ever.

Reply:
I've added the Kay et al. 2015 reference to the CESM-LE in the metadata. I think the metadata can be very descriptive as well as the Zenodo entry. I believe we should keep the filename simple.